projects/packages/search/src/class-stats.php

```diff
@@ -34,4 +38,52 @@ public function get_stats_from_wpcom() {
 
 		return $response;
 	}
+
+	/**
+	 * Estimate record counts via a local database query.
+	 */
+	public static function estimate_count() {
+		$cached_value = wp_cache_get( self::COUNT_ESTIMATE_CACHE_KEY, self::CACHE_GROUP );
+		if ( false !== $cached_value ) {
+			return $cached_value;
+		}
+
+		global $wpdb;
+		$indexable_stati        = get_post_stati( array( 'public' => true ) );
```

Comment:
I think the correct plural is `statuses` but WordPress core uses `stati` in this case 🤔 Would be tempted to use `statuses` in our code.

https://www.merriam-webster.com/dictionary/status

Reply:
It is "hypercorrect" https://en.wiktionary.org/wiki/stati#Noun https://en.wiktionary.org/wiki/Appendix:Glossary#hypercorrect The misapplication of an ending rule due to where the word was appropriated from (Latin vs Greek vs German). Personally I do it all the time. :)

Reply:
TIL

Reply:
Adjusted to use `statuses`.

Reply:
TIL about hypercorrect (and octopodes 🐙🐙🐙), thanks @gibrown!